README.md

```diff
@@ -1,4 +1,13 @@
-# photon
+<h1 align="center" style="border-bottom: none;">🚀 Nano / Photon </h1>
+<h3 align="center">Theme-able component library</h3>
+<p align="center">
+  <a href="https://v2.vuejs.org/">
+    <img alt="Join the community on GitHub Discussions" src="https://img.shields.io/badge/Vue.js-2.6.11-4fc08d?logo=Vue.js">
+  </a>
+  <a href="https://github.com/semantic-release/semantic-release/">
+    <img alt="semantic-release: angular" src="https://img.shields.io/badge/semantic--release-angular-e10079?logo=semantic-release">
```

Comment:
angular semantic release?

Reply:
The conventions (the commit messages) are the angular conventions.

As mentioned in the conventional-commits page


> types other than fix: and feat: are allowed, for example [@commitlint/config-conventional](https://github.com/conventional-changelog/commitlint/tree/master/%40commitlint/config-conventional) (based on the [the Angular convention](https://github.com/angular/angular/blob/22b96b9/CONTRIBUTING.md#-commit-message-guidelines)) recommends build:, chore:, ci:, docs:, style:, refactor:, perf:, test:, and others.


 
https://github.com/angular/angular/blob/22b96b9/CONTRIBUTING.md#-commit-message-guidelines